exercises/practice/queen-attack/queen_attack_test.c

```diff
@@ -0,0 +1,97 @@
+// Version: 1.0.0
+
+#include "vendor/unity.h"
+
+// Returns 1 if queen can be created, otherwise 0.
+extern int can_create(int row, int column);
```

Comment:
Two possible alternatives:

typedef struct {
   int64_t row;
   int64_t column;
} position_t;

Alternative One:

// Populates and returns queen.
// Returns 0 if row or column are invalid.
position_t* create(int64_t row, int64_t column, position_t* queen);

Alternative Two (from C track):

typedef enum { CAN_NOT_ATTACK, CAN_ATTACK, INVALID_POSITION } attack_status_t;

// create function omitted.

attack_status_t can_attack(position_t queen_1, position_t queen_2);